feelpp/feel/feelvf/detail/ginacmatrix.hpp

```diff
@@ -1125,7 +1153,7 @@ private :
         if ( M_isNumericExpression )
             return;
 
-        std::vector<std::pair<GiNaC::symbol,int>> symbTotalDegree;
+        std::vector<std::pair<GiNaC::symbol,uint16_type>> symbTotalDegree;
```

Comment:
warning could you keep int ? 
I am making in another branch the polynomial degree dynamic like in Eigen (compile time or dynamic)

Reply:
here is a discussion also on uint vs int http://eigen.tuxfamily.org/index.php?title=FAQ#Why_Eigen.27s_API_is_using_signed_integers_for_sizes.2C_indices.2C_etc..3F

Reply:
I am not sure to understand why it's a problem because, here, it's a local variable used in the algorithm (and I need to check if value is invalid_v< uint16_type> or not. If really an issue, we can change to int and check non negative value.